scripts/typecheck_tests.py

```diff
@@ -188,8 +188,6 @@
         "base class",
         '"CharField" has incompatible type "Collection[Any]"',
         'Name "foo" is not defined',
-        'Unsupported left operand type for | ("ReturnDict")',
-        'Unsupported left operand type for | ("Dict[str, str]")',
```

Comment:
You can't remove these, they are still necessary for older Python versions.

Reply:
ah, I didn't realize that -- thanks!